images/virtualization-artifact/pkg/controller/vmip/internal/iplease_handler.go

```diff
@@ -63,9 +63,10 @@ func (h IPLeaseHandler) Handle(ctx context.Context, state state.VMIPState) (reco
 	if err != nil {
 		return reconcile.Result{}, err
 	}
+	condition, _ := service.GetCondition(vmipcondition.BoundType, vmipStatus.Conditions)
 
 	switch {
-	case lease == nil && vmipStatus.Address != "":
+	case lease == nil && vmipStatus.Address != "" && condition.Reason != vmipcondition.VirtualMachineIPAddressLeaseAlready:
```

Comment:
Rename reason `VirtualMachineIPAddressLeaseAlready`. It's not clear: already what?

Reply:
Fixed